frontend/src/Components/CreateLendPage.tsx

```diff
@@ -0,0 +1,224 @@
+import { Box, Container, Paper, Typography, TextField, Button, Table, TableBody, TableCell, TableRow} from "@mui/material";
+import { useEffect, useState } from "react";
+import { Navigate, useNavigate } from "react-router-dom";
+import { ROUTER_PATHS } from "../Constants";
+import { ItemType } from "../Types/types";
+
+
+const defaultItem: ItemType = {
+	id: -1,
+	name: '',
+	lend_start: '',
+	lend_end: '',
+	img_uri: '',
+	is_lent_item: true
+}
+
+const defaultFormValues = {
+	name: '',
+	days: ''
+}
+
+const CreateLendPage = () => {
+
+	const navigation: any = useNavigate();
+
+	let [formValues, setFormValues] = useState(defaultFormValues)
+	let [item, setItem] = useState(defaultItem)
+
+	let [isCreated, setIsCreated] = useState(false)
+	let [title, setTitle] = useState("Create a Lend Item")
+	let [startIndex, setStartIndex] = useState(0)
+	let [endIndex, setEndIndex] = useState(0)
```

Comment:
These could all be set into a map for all the states. Might be a bit cleaner, but less readable. Up to you, really.

Reply:
How would you go about creating a map? I'm not sure I know how to do that.

Reply:
It's pretty much like a dictionary/json type format.
something like let [state, setState] = useState({
created: false,
start_idx: 0,
end_idx: 0
})
Pretty much just grouping everything. But its not a neccesary change forsure.

Reply:
Ohhhhhhhhh gotcha. Ok. Even if I don't do that here, that's a really good idea I'm going to use at some point.